plugins/wordle/gamehandler.py

```diff
@@ -193,8 +203,8 @@ async def on_message(self, msg: Message):
         assert len(response) == WORDLENGTH
         for i in range(len(response)):
             found = False
-            for correctness, icon in ICONS.items():
-                if response[i] == icon:
+            for correctness, icon in self.ui.items():
+                if response[i] in icon:
```

Comment:
```suggestion
                if response[i].lower() in icon:
```
oder halt oben (L174) manuell einfügen.

Reply:
Also halt, dass auch großgeschrieben die akzeptiert werden!